exampleSite/content/footer.md

```diff
@@ -0,0 +1,19 @@
++++
+fragment = "footer"
+#disabled = true
+date = "2016-09-07"
+lastmod = "2017-09-07"
+weight = 1200
+#background = ""
+
+linkTitle = "Link Title"
+logoTitle = "Logo Title"
+logo = "logo.svg"
+logoText = "Logo Subtext"
+descriptionTitle = "Description Title"
+description = """
```

Comment:
description could be moved to the content section. So that it can be accessed via .Content and is not part of the frontmatter. That would make markdown enabled content easier to format.